packages/template-retail-react-app/app/pages/cart/index.test.js

```diff
@@ -1229,85 +1230,51 @@ describe('Product bundles', () => {
     })
 })
 
-describe('Unavailable products tests', function () {
-    test('Remove unavailable/out of stock/low stock products from cart', async () => {
-        prependHandlersToServer([
-            {path: '*/customers/:customerId/baskets', res: () => mockBaskets},
-            {path: '*/products', res: () => mockProducts}
-        ])
+describe('Bonus products', () => {
+    beforeEach(() => {
+        global.server.use(
```

Comment:
How about using `prependHandlersToServer` to reduce the boilerplate code for api mock?